contracts/stake-cw20-external-rewards/src/contract.rs

```diff
@@ -1993,9 +1978,11 @@ mod tests {
         let scaled_min = Uint256::from(min_128).checked_mul(scale_factor).unwrap();
         println!("scaled {}", scaled_min);
         println!("scaled {}", scaled_min.to_string().len());
-        assert_eq!(scaled_min>Uint256::from(Uint128::MAX), true);
+        assert!(scaled_min > Uint256::from(Uint128::MAX));
 
-        let max = Uint256::from(Uint128::MAX).checked_mul(scale_factor).unwrap_err();
+        let _max = Uint256::from(Uint128::MAX)
+            .checked_mul(scale_factor)
+            .unwrap_err();
 
         let possible_max = Uint256::MAX.checked_div(scale_factor).unwrap();
         println!("possible max: {}", possible_max);
```

Comment:
```suggestion
```

Reply:
This is here just for debugging purposes. Don't see a need to remove it